_common/micro/MicroService.js

```diff
@@ -117,6 +117,12 @@ MicroService.prototype.getSystemCodes = function (next) {
       }
 
       global.systemCodes = systemCodes;
+
+      var clusterTypeSystemCode = !_.isNaN(parseInt(
```

Comment:
We need to add a check for this in app.js. We cannot set globals other than `global.systemCodes` as everything else is defined in `setupMS`. It should be quite easy to check if the node is restricted whenever needed.